py/packages/genkit/src/genkit/ai/model.py

```diff
@@ -2,7 +2,13 @@
 # SPDX-License-Identifier: Apache-2.0
 
 from collections.abc import Callable
+from pydantic import BaseModel
+from genkit.core.schemas import GenerateRequest, GenerateResponse, ModelInfo
 
-from genkit.core.schemas import GenerateRequest, GenerateResponse
 
 ModelFn = Callable[[GenerateRequest], GenerateResponse]
+
+
+class ModelReference(BaseModel):
+  name: str
+  info: ModelInfo
```

Comment:
model reference should also contain the config
https://github.com/firebase/genkit/blob/90e23213e4bb0c15e453dfa6db20d32d8ad2d9ed/js/ai/src/model.ts#L545

Reply:
no need to add right away, could you please just add a todo?